Domain/Services/Implementations/PostService.cs

```diff
@@ -49,4 +49,17 @@ public Post GetPost(Guid guid)
 
         return post;
     }
+
+    public List<Post> GetUsersPosts(Guid guid)
+    {
+        User? user = uow.Repository<User>().Get(guid);
+        if (user is null)
+            throw new InvalidGuidException<User>();
+
+        List<Post> posts = new List<Post>();
+        foreach (var e in user.SubscribedEvents.AsEnumerable())
+            posts.AddRange(e.Posts);
+
+        return posts;
+    }
```

Comment:
First of all I would suggest adding any authorization from BaseService. Secondly most of service methods return IEnumerable. It doesn't change much but I would stick to it. Also we could get user from ActingUser, but that depends on what is intended (check my comment in PostsController).
```suggestion

    public IEnumerable<Post> GetUsersPosts()
    {
        AllowOnlyLoggedIn();

        List<Post> posts = new List<Post>();
        foreach (var e in ActingUser!.SubscribedEvents.AsEnumerable())
            posts.AddRange(e.Posts);

        return posts;
    }
```